ccm_web/server/src/canvas/canvas.service.ts

```diff
@@ -108,4 +109,12 @@ export class CanvasService {
     const token = user.canvasToken === undefined ? null : user.canvasToken
     return token
   }
+
+  async createRequestorForUser (userLoginId: string): Promise<CanvasRequestor> {
+    const token = await this.findToken(userLoginId)
+    if (token === null) throw new Error(`User ${userLoginId} does not have a token!`)
+
+    const requestor = new CanvasRequestor(this.url + '/api/v1/', token.accessToken)
```

Comment:
Anyone see a reason to not hardcode `/api/v1`, make it configurable?

Reply:
Currently all apis are on` /api/v1 ` **except** graphql which is just on `/api/graphql`. Canvas might eventually have some future API's on `/api/v2`, `/api/v3` etc if the API ever changed and they maintained the old API. 

You might just want to make that `/api/v1/` as a method parameter with this value as the default.
 
We removed this hardcoding in the canvasapi Python library recently.

Reply:
Good idea, how is this, @jonespm? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/121/commits/39477d2334738f5618a6348a191729adb03d56cf